aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -231,8 +246,8 @@ module aptos_framework::account {
                 guid_creation_num,
                 coin_register_events,
                 key_rotation_events,
-                rotation_capability_offer: CapabilityOffer { for: option::none() },
-                signer_capability_offer: CapabilityOffer { for: option::none() },
+                rotation_capability_offer: CapabilityOffer {for: option::none()},
+                signer_capability_offer: CapabilityOffer {for: option::none()},
```

Comment:
No spaces?

Reply:
we will fix it

Reply:
fixed at https://github.com/movebit/movefmt/commit/a502614c0de751126eae46f2dcf4c90d37a8faee